types/plugin.go

```diff
@@ -29,7 +29,7 @@ type Plugin struct {
 	ID       string `json:"Id,omitempty"`
 	Name     string
 	Tag      string
-	Active   bool
+	Enabled  bool
```

Comment:
I know we don't do this in this project, but could we _please_ document our fields? What does enabled actually mean?